Applications/FileIO/GocadIO/GocadAsciiReader.cpp

```diff
@@ -452,7 +614,7 @@ bool GocadTSurfaceReader::parseElements(
     return false;
 }
 
-bool GocadTSurfaceReader::skipModel3d(std::ifstream& in) const
+bool GocadAsciiReader::skipToEND(std::ifstream& in) const
```

Comment:
```suggestion
bool GocadAsciiReader::skipToEnd(std::ifstream& in) const
```

Reply:
I used this notation because the tag in the gocad-file is actually called ```END```. If it's called "skipToEnd" it suggests that it jumps to the end of the file?